### ISSUE TITLE
Discovery: Update "Adding a new feature" page

### ISSUE THREAD

**Issue by BSmolensky** [department-of-veterans-affairs/va.gov-team#74464]
The update we made to [Creating a new content page](https://depo-platform-documentation.scrollhelp.site/developer-docs/creating-a-new-content-page) involved removing all references to `vagov-content`as Drupal is now the primary source of content. 

On the Creating a new content page, the user finds a link to [Adding a new feature](https://depo-platform-documentation.scrollhelp.site/developer-docs/Adding-applications-and-features.1847591039.html). The first section of that page has the following language in it: 

> Typically these pages are added by content editors in the vagov-content repo or in the Drupal CMS.

So, the page we were working on links to a page that contains precisely the information we were deleting on the Create a New Content Page.

We checked with the SME (Erika Washburn) and she was unsure if that verbiage needed to be changed, saying in part, "Some folks still use the `vagov-content` and apparently there might be a new team over that specific repo that I can point you to. I believe that is the IIR Team- Innovate, Iterate, and Run Team." 

Her recommendation was to create a ticket, put it in our backlog, work on it during another sprint. This is that card :) 
## Slack Thread - most recent re: next steps
https://dsva.slack.com/archives/C05RJS5DANT/p1718396060050739

### TASKS

- [x] Circle back with Erika Washburn if necessary
- [x] Find the appropriate SME (most likely the IIR Team)
- [x] Get a thumbs up or thumb down on the content as it is written on the page currently.
- [x] If changes are needed, create a new ticket for the changes. 
- [ ] If no changes are needed, close this ticket and relax. 

## SME:
Nick Sayre

### ACCEPTANCE CRITERIA

- [ ] A determination has been made about the content currently on the [Adding a new feature](https://depo-platform-documentation.scrollhelp.site/developer-docs/Adding-applications-and-features.1847591039.html) page.

**Comment by jknipes** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Content-Team-62bde00be46b400017674aef/issues/department-of-veterans-affairs/va.gov-team/74464?planning-poker) with Zenhub @BSmolensky @CherylEvans @laucon @taylojill

**Comment by jknipes** [comment]
I pinged the IIR team [in Slack](https://dsva.slack.com/archives/CBU0KDSB1/p1718395391588039?thread_ts=1693432963.478369&cid=CBU0KDSB1) to see if they would be able to review the page.

**Comment by jknipes** [comment]
@nicksayre

**Comment by jknipes** [comment]
cc: @nicksayre

**Comment by taylojill** [comment]
I updated the Adding applications and features page, see summary below. I sent pdf copy of changes to Nick Sayre (SME) via platform content channel.
-  Added a blue info box stating the vagov-content repo is still being used by several teams, and 
-  I added a link to the GitHub Readme file provided by SME. 
-  I reworded one sentence on the page to clarify that Drupal is now the primary source of content (blue color font).
